### ISSUE TITLE
hangman|";ls -la #

### ISSUE THREAD

**Issue by g4mm4** [roveroniandrea/roveroniandrea#46]
the last issue; just 4 fun ;) sorry 4 any inconveniences

**Comment by roveroniandrea** [comment]
I think you did it!

**Comment by g4mm4** [comment]
Yeah, pretty sure, the first request is submitted by my github bot
@roveroniandrea did you see that RCE issue ;) ?

**Comment by roveroniandrea** [comment]
@g4mm4 Really well done, i saw it! I think the github action should escape the issue name, or maybe check for some special characters... Anyway this was fun!

**Comment by g4mm4** [comment]
@roveroniandrea check this:
https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#understanding-the-risk-of-script-injections
